installers/kubectl/client-setup.sh

```diff
@@ -78,9 +78,14 @@ kubectl_get_private "${OUTPUT_DIR}/pgouser" secret -n "${PGO_OPERATOR_NAMESPACE}
 kubectl_get_private "${OUTPUT_DIR}/client.crt" secret -n "${PGO_OPERATOR_NAMESPACE}" pgo.tls -o 'go-template={{ index .data "tls.crt" | base64decode }}'
 kubectl_get_private "${OUTPUT_DIR}/client.key" secret -n "${PGO_OPERATOR_NAMESPACE}" pgo.tls -o 'go-template={{ index .data "tls.key" | base64decode }}'
 
+client_vars=$(cat <<EOF
+export "PATH=${OUTPUT_DIR}:\$PATH"
+export "PGOUSER=${OUTPUT_DIR}/pgouser"
+export "PGO_CA_CERT=${OUTPUT_DIR}/client.crt"
+export "PGO_CLIENT_CERT=${OUTPUT_DIR}/client.crt"
+export "PGO_CLIENT_KEY=${OUTPUT_DIR}/client.key"
+EOF
+)
+
 echo "pgo client files have been generated, please add the following to your bashrc"
-echo "export PATH=${OUTPUT_DIR}:\$PATH"
-echo "export PGOUSER=${OUTPUT_DIR}/pgouser"
-echo "export PGO_CA_CERT=${OUTPUT_DIR}/client.crt"
-echo "export PGO_CLIENT_CERT=${OUTPUT_DIR}/client.crt"
-echo "export PGO_CLIENT_KEY=${OUTPUT_DIR}/client.key"
+echo "$client_vars" | tee ${OUTPUT_DIR}/client_vars.rc
```

Comment:
No need for `tee` here, e.g. can use 

```bash
echo "$client_vars" > ${OUTPUT_DIR}/client_vars.rc`
```

Reply:
I used `tee` so that the lines would still be printed to `stdout` **and** sent to the file. That way I'm not changing the current usage of the file but still adding the new functionality.

If I change how the script works then I will need to go in and update docs related to this script. I can do that but I was trying to limit this change.

Reply:
Another option (without `tee`)
```
echo "$client_vars"
echo "$client_vars" > ${OUTPUT_DIR}/client_vars.rc
```

Reply:
Cool, that definitely makes sense.  Appreciate the explanation!